### ISSUE TITLE
Ratelimits.

### ISSUE THREAD

**Issue by Dhravya** [Dhravya/dhravyaAPI#31]
Ratelimits are extremely important and we need to have some sort of rate limits.

**Comment by PolyAnthi** [comment]
This can come in the near future when we grow alot imo.

**Comment by Dhravya** [comment]
no, the issue is someone can just decide to send a billion requests in a second and we would have nothing to do about it

**Comment by PolyAnthi** [comment]
Not necessarily as 1) thats impossible, 2) we would notice, 3) why would
anyone do that

**Comment by Dhravya** [comment]
@itsnotrin Don't you think ratelimits are important tho?

**Comment by itsnotrin** [comment]
We need them.

**Comment by Dhravya** [comment]
alright, i checked the notification pretty late

Just mention the limits here and i'll implement it

**Comment by itsnotrin** [comment]
I say we have per endpoint rate limits

**Comment by Dhravya** [comment]
yeah ofc, per endpoint limits make the most sense
